publication-commons/src/main/java/no/unit/nva/publication/doi/dto/Publication.java

```diff
@@ -0,0 +1,132 @@
+package no.unit.nva.publication.doi.dto;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.net.URI;
+import java.util.List;
+import nva.commons.utils.JacocoGenerated;
+
+public class Publication {
```

Comment:
Shouldn't this DTO have a constructor from the DAO ?

Reply:
No, this is DTO only for `publication/doi`, and it is self contained for that purpose. 

https://unit.atlassian.net/browse/NP-1704 will build a mapper to create this DTO from Publication dynamodb stream event.